### PR TITLE
feat(perf): Performance Database View

### PR DESCRIPTION
**Pull request by gggritso**
Documentation for the V0 release of the Database View in Performance.

### CODE REVIEW

File: src/docs/product/performance/query-insights.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+
+The data shown is pulled from `db` and `db.sql` spans.
+
+Sentry tries to extract metrics for all SQL-like dialects. NoSQL databases like MongoDB, graph databases like Neo4j, and other non-SQL database systems are not eligible for this feature.
+
+If you are using <PlatformLink to="/performance/instrumentation/automatic-instrumentation">automatic instrumentation</PlatformLink>, Query Insights should work without any configuration. If you've manually instrumented Sentry, you'll need to make sure that your spans conform to our standards for the best experience:
+
+- The span `op` field is set to an [eligible value](https://develop.sentry.dev/sdk/performance/span-operations/#database).
+- The span's description contains the full SQL query.
+- The `db.system` span data value is set to the correct identifier (e.g., `"postgresql"` or `"mysql"`).
```
Comment (AbhiPrasad) [inline src/docs/product/performance/query-insights.md:52]: This matches https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/database.md#notes-and-well-known-identifiers-for-dbsystem - do we want to link there?

File: src/docs/product/performance/query-insights.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+- [Java SDK v6.29.0](https://github.com/getsentry/sentry-java/releases/tag/6.29.0)
+- [Ruby SDK v5.11.0](https://github.com/getsentry/sentry-ruby/releases/tag/5.11.0)
+- [.NET SDK v3.39.0](https://github.com/getsentry/sentry-dotnet/releases/tag/3.39.0)
+- [Symfony SDK v4.11.0](https://github.com/getsentry/sentry-symfony/releases/tag/4.11.0)
+
+### Span Eligibility
+
+The data shown is pulled from `db` and `db.sql` spans.
+
+Sentry tries to extract metrics for all SQL-like dialects. NoSQL databases like MongoDB, graph databases like Neo4j, and other non-SQL database systems are not eligible for this feature.
```
Comment (mwarkentin) [inline src/docs/product/performance/query-insights.md:46]: Should we say "not currently eligible" instead? "not currently supported"?

**Review by jjbayer — APPROVED** [review]
Left one suggestion, everything else LGTM!

File: src/docs/product/performance/query-insights.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+
+</br>
+
+### Query Parameterization
+
+In some cases, Sentry processes queries and simplifies them to improve readability. For example:
+
+- Table names are removed from column selections if possible (e.g., `SELECT "users"."name", "users"."id", "users.age" FROM users` becomes `SELECT name, id, age FROM users`)
+- Long lists of `SELECT` arguments are collapsed (e.g., `SELECT id, name, age, city, country, phone FROM users` becomes `SELECT .. FROM users`)
+- Long lists of `VALUES` arguments are collapsed (e.g., `INSERT INTO users VALUES (%s %s %s %s), (%s %s %s %s)` becomes `INSERT INTO users VALUES ..`)
```
[the remark this answers is not in the document]
Comment (gggritso) [reply src/docs/product/performance/query-insights.md]: Thanks!

**Pull request merged**